### ISSUE TITLE
undefined method `message_verifier' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by mytrile** [rails/activemodel-globalid#3]
Fresh rails 4.1.1 app with ruby 2.1.1p76 (2014-02-24 revision 45161) [x86_64-darwin13.0]

``` sh
$ rails c

/Users/dimitar/.gem/ruby/2.1.1/gems/activemodel-globalid-0.1.0/lib/active_model/railtie.rb:3:in `<top (required)>': undefined method `message_verifier' for nil:NilClass (NoMethodError)
    from /Users/dimitar/.gem/ruby/2.1.1/gems/activerecord-4.1.1/lib/active_record/railtie.rb:3:in `require'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/activerecord-4.1.1/lib/active_record/railtie.rb:3:in `<top (required)>'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/railties-4.1.1/lib/rails/all.rb:12:in `require'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/railties-4.1.1/lib/rails/all.rb:12:in `block in <top (required)>'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/railties-4.1.1/lib/rails/all.rb:10:in `each'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/railties-4.1.1/lib/rails/all.rb:10:in `<top (required)>'
    from /Users/dimitar/code/rails/demo/config/application.rb:3:in `require'
    from /Users/dimitar/code/rails/demo/config/application.rb:3:in `<top (required)>'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application.rb:82:in `require'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application.rb:82:in `preload'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application.rb:140:in `serve'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application.rb:128:in `block in run'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application.rb:122:in `loop'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application.rb:122:in `run'
    from /Users/dimitar/.gem/ruby/2.1.1/gems/spring-1.1.3/lib/spring/application/boot.rb:18:in `<top (required)>'
    from /Users/dimitar/.rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/dimitar/.rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from -e:1:in `<main>'
```

Without the gem:

``` irb
$ rails c
Loading development environment (Rails 4.1.1)
irb(main):001:0> Rails.application.message_verifier(:signed_global_ids)
=> #<ActiveSupport::MessageVerifier:0x007f8100c8ab08 @secret="=\xC9q\xDB9\xCEKb\xE9\xC7\xFA\xEBD\x12\xA2\x96-\xAC\x1A,\x16\x12\xA7\xDCb\xBF \xB5\xAF\xA6GO5\xDF\xD3b\xE5(\xC6\x84\x1D\xC4^\x7F\xAE\xD9\xCC\xFE1\x7F\xD0O\xC5/v]im'\x1FC\x00\xD6^", @digest="SHA1", @serializer=Marshal>
```

EDIT:

It seems like Rails.application is not available outside initializer block or something

**Comment by ericboehs** [comment]
Is a new gem going to be released for this? 0.1.0 was released a day before this commit landed. Active job doesn't seem to work with Rails 4.1 out of the box.

**Comment by dhh** [comment]
Pushed a new gem.
